### PR TITLE
Run make gen to sync charts

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ metadata:
       The Maistra Operator enables you to install, configure, and manage an instance of Maistra service mesh. Maistra is based on the open source Istio project.
 
     containerImage: maistra/istio-ubi8-operator:2.1.0
-    createdAt: 2021-03-30T10:38:31CEST
+    createdAt: 2021-06-16T11:22:14EDT
```
Comment (shamsher31) [inline manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml]: Question: Is this useful anywhere? It just keeps changing the timezone.

**Review by shamsher31 — APPROVED** [review]

File: manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (jwendell) [reply manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml]: I'll revert this change

Comment (jwendell) [reply manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml]: done. I hate this btw.

**Review by shamsher31 — COMMENTED** [review]
Can we write some kind of automation that will update the charts whenever a new chart update is available so we don't have to manually create PR to update the chart?

**Comment by jwendell** [comment]
> Can we write some kind of automation that will update the charts whenever a new chart update is available so we don't have to manually create PR to update the chart?

I think so yes. There are a few tricks like updating the tests. Certainly nothing impossible. Feel free to create a ticket for it so we can work on it next sprint.

**Review by rcernich — APPROVED** [review]

**Pull request merged**